src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Calculator/CalculateHelper.cs

```diff
@@ -60,6 +64,37 @@ public static string FixHumanMultiplicationExpressions(string input)
             return output;
         }
 
+        private static string CheckScientificNotation(string input)
+        {
+            /**
+             * NOTE: Anywhere you see a "{0}", that is meant to be replaced by the user's decimal separator.
+             *
+             * Regex explanation:
+             * (-?(\d+({0}\d*)?)|-?({0}\d+)): Used to capture one of two types:
+             * -?(\d+({0}\d*)?): Captures a decimal number starting with a number (e.g. "-1.23")
+             * -?({0}\d+): Captures a decimal number without leading number (e.g. ".23")
+             * E: Captures capital 'E'
+             * (-?\d+{0}?\d*): Captures decimal number starting with a number (e.g. "1.23")
+             */
+            var p = @"(-?(\d+({0}\d*)?)|-?({0}\d+))E(-?\d+{0}?\d*)";
```

Comment:
can we have a decimal point after E? 🤔

Reply:
i.e. should it be 
var p = @"(-?(\d+({0}\d*)?)|-?({0}\d+))E(-?\d+)";

Reply:
Nope. The following number must be an integer.

Reply:
Yeah, I went back and forth on this one and initially decided to allow users to do something like `1E1.2` if they really really wanted to do `1 * 10^(1.2)`, but it's definitely not the intended usage of the notation. I removed it so that it only allows integers after the `E` in my latest push